src/components/Search/CompareWithBase.tsx

```diff
@@ -104,9 +104,11 @@ function CompareWithBase({
     (baseRepository === 'try' && newRepository !== 'try') ||
     (baseRepository !== 'try' && newRepository === 'try');
 
-  const possiblyPreventFormSubmission = (e: React.FormEvent) => {
+  const onFormSubmit = (e: React.FormEvent) => {
     const isFormReadyToBeSubmitted = baseInProgressRev !== null;
+    setFormIsDisplayed(false);
     if (!isFormReadyToBeSubmitted) {
+      setFormIsDisplayed(true);
```

Comment:
Had to set to `true` here because a Search View test breaks - the forms are hidden.

Reply:
What about using this line 109:
```
setFormIsDisplayed(!isFormReadyToBeSubmitted)
```
so that you don't need to set it twice?

Reply:
Ahh right! lol